lib/matplotlib/axis.py

```diff
@@ -708,6 +722,9 @@ def __init__(self, axes, pickradius=15):
         self.cla()
         self._set_scale('linear')
 
+    majorTicks = _LazyTickList(major=True)
```

Comment:
OK, I've never seen descriptors, so I looked them up, and this is fine, but maybe for the uninitiated you could say: "# use a class descriptor for major/minorTicks to allow lazy instantiation"

Reply:
done

Reply:
The descriptor is really a neat way of lazy instantiation! Could you please add a short class docstring along the lines of "A descriptor for a lazy on-demand instantiated tick list."? Looks good otherwise.

Reply:
done